Authoring/AuthoringSample/AuthoringSample.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <LangVersion>preview</LangVersion>
+    <AssemblyVersion>1.0.0.0</AssemblyVersion>
+    <EnableCsWinRTLogging>true</EnableCsWinRTLogging>

```

Comment:
do you expect to make this permanent?  Can we rename to CsWinRTEnableLogging?

Reply:
Yea I think we are going to need some form of logging mode.  I can rename it.